.github/workflows/release_dataops_cloudbeaver.yml

```diff
@@ -0,0 +1,59 @@
+name: Build & Publish DataOps CloudBeaver Image
+
+on:
+  push:
+    tags:
+      - 'dataops-cloudbeaver-*.*.*'
+
+jobs:
+  buildx-dataops:
+    runs-on: ubuntu-latest
```

Comment:
I think there is missing matrix
```
    strategy:
      matrix:
        provider: ["bigquery"]
```

Reference:
https://github.com/getindata/jupyter-images/blob/jupyterlab-dataops-1.0.6/.github/workflows/release_dataops.yml#L11

Reply:
That's actually a question if we should build CloudBeaver for non-bigquery targets as well. For now it's need to build and publish CB images as there is no other way to simply attach BigQuery driver to the application.

Reply:
For now dbeaver image with bigquery can only be used for bigquery or we only add a new driver, and the user can connect to any driver? If yes wouldn't be better to have only one reference to dbeaver image in this dockerfile?

Reply:
I left some drivers [you can check here](https://github.com/getindata/jupyter-images/pull/35/files/8703eb6907d10b25ebfeffff1ad9dd852f340ed1#diff-0528e116b6e8d61cc476bc89eb5d7bf17f8804365a5fbbeb83bb1c034d92a145R14)

How I imagined build process was we build a broader matrix of providers, like Redshift/Snowflake/etc. and for those we add the driver as well. Otherwise we use default CloudBeaver image from dockerhub

Reply:
How image size would change leaving all drivers and adding our custom for bigquery? There will be some drawbacks rather than splitting images into different drivers?

Reply:
It's a matter of couple of megabytes. 
That's a [list](https://github.com/dbeaver/cloudbeaver/blob/devel/server/drivers/pom.xml) of default drivers installed. Anything you consider reduntant?

Note that we can also build image with more drivers than disable some of them in config.

Reply:
What do you think about leaving the default drivers and add only for bigquery, if it's only a couple of megabytes I think it's a bigger overhead splitting images default/bigquery and in future, many customs rather than have only one image that will contain all drivers. And if the number of drivers in UI is overwhelming we can disable them as you say.